Assets/Scripts/UI/Menu's/PauseController.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
+
+public sealed class PauseController : MenuController
+{
+    [SerializeField] private GameObject activePauseMenu;
+    [SerializeField] private GameObject activePauseScreen;
+    [SerializeField] private GameObject activeOptionsScreen;
```

Comment:
```suggestion
    [SerializeField] private GameObject parentPauseMenu;
    [SerializeField] private GameObject pauseScreen;
    [SerializeField] private GameObject optionsScreen;
```
Zelfde feedback als feedback 1. Ook zal ik de main pause object de naam parent geven voor meer duidelijkheid, want die 2 varibels hun naam lijken veel op elkaar.